explore/src/main/scala/explore/config/TargetImportPopup.scala

```diff
@@ -128,14 +162,27 @@ object TargetImportPopup:
             ProgressSpinner(strokeWidth = "5").unless(state.value.done),
             Icons.Checkmark.withSize(IconSize.X4).when(state.value.done),
             <.div(
+              ExploreStyles.TargetImportDescription,
               <.span(s"Importing ${state.value.loaded.length}")
                 .unless(state.value.done),
               <.span(s"Imported ${state.value.loaded.length} targets").when(state.value.done),
-              <.span(s"Import errors: ${state.value.targetErrors}")
+              <.span(s"Import errors: ${state.value.targetErrors.length}")
             ).when(state.value.genericError.isEmpty),
             <.div(
+              ExploreStyles.TargetImportDescription,
               <.span(s"Import error: ${state.value.genericError.orEmpty}")
-            ).unless(state.value.genericError.isEmpty)
+            ).unless(state.value.genericError.isEmpty),
+            <.div(
+              ExploreStyles.TargetImportErrors |+| ExploreStyles.ExploreBorderTable,
+              PrimeAutoHeightVirtualizedTable(
+                table,
+                estimateRowHeight = _ => 34.toPx,
+                tableMod = ExploreStyles.ExploreTable,
+                striped = true,
+                compact = Compact.Very,
+                emptyMessage = "No lines defined"
```

Comment:
Is this the right emptyMessage for this table?

Reply:
Good catch, will fix it